flash/tabular/classification/input.py

```diff
@@ -65,3 +65,31 @@ def load_data(
             return super().load_data(
                 read_csv(file), categorical_fields, numerical_fields, target_fields, parameters, target_formatter
             )
+
+
+class TabularClassificationListInput(TabularDataFrameInput, ClassificationInputMixin):
+    def load_data(
+        self,
+        data: Dict[str, Union[Any, List[Any]]],
+        categorical_fields: Optional[Union[str, List[str]]] = None,
+        numerical_fields: Optional[Union[str, List[str]]] = None,
+        target_fields: Optional[Union[str, List[str]]] = None,
+        parameters: Dict[str, Any] = None,
+        target_formatter: Optional[TargetFormatter] = None,
+    ):
+        # Convert the data (dict) to a Pandas DataFrame
+        data_frame = DataFrame.from_dict(data)
+
+        cat_vars, num_vars = self.preprocess(data_frame, categorical_fields, numerical_fields, parameters)
+
+        if not self.predicting:
+            targets = resolve_targets(data_frame, target_fields)
+            self.load_target_metadata(targets, target_formatter=target_formatter)
+            return [{DataKeys.INPUT: (c, n), DataKeys.TARGET: t} for c, n, t in zip(cat_vars, num_vars, targets)]
+        else:
+            return [{DataKeys.INPUT: (c, n)} for c, n in zip(cat_vars, num_vars)]
+
+    def load_sample(self, sample: Dict[str, Any]) -> Any:
+        if DataKeys.TARGET in sample:
+            sample[DataKeys.TARGET] = self.format_target(sample[DataKeys.TARGET])
+        return sample
```

Comment:
This shouldn't be needed if it's defined in the super class

Reply:
Fixed! :)